src-ext/Typescript/ggschema/src/scale.ts

```diff
@@ -0,0 +1,5 @@
+export interface Scale {
+  aesthetics: string[];
+  transform: string;
```

Comment:
I think `transform` needs to be an object, see <https://github.com/vegawidget/ggvega/blob/master/dev/exploratory/iris-scatterplot-01.json>

Reply:
Note for future reference - no action necessary now - it would not surprise me that the ggscpec definition of scales will expand in the not-so-distant future.